ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/store/dao/JdbcLedgerDaoBackendH2Database.scala

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.store.dao
+
+import com.digitalasset.platform.store.DbType
+import org.scalatest.Suite
+
+private[dao] trait JdbcLedgerDaoBackendH2Database extends JdbcLedgerDaoBackend { this: Suite =>
```

Comment:
Why make this (and `JdbcLedgerDaoBackendPostgresql`) a trait rather than adding the overrides directly to the spec classes?

Reply:
Leaving the spec classes as simple as they can, make files as small as possible. I don't see a big advantage in shifting code there to be honest.